File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/ord/pub/d/AllergicPharmHandleMode.java
package iih.ci.ord.pub.d;

import xap.sys.appfw.orm.desc.ent.dataobject.DmEnumDesc;

public interface AllergicPharmHandleMode {

    @DmEnumDesc(name="过敏用药禁用",description="过敏用药禁用")
	public static final Integer ALLERGICPHARMFORBIDDEN=0;
    @DmEnumDesc(name="过敏再皮试",description="过敏再皮试")
	public static final Integer ALLERGICRESKINTEST=1;
    @DmEnumDesc(name="皮肤过敏试验",description="皮肤过敏试验")
	public static final Integer SKINALLERGICTEST=3;
    @DmEnumDesc(name="过敏强制使用",description="过敏强制使用")
	public static final Integer ALLERGICFORCEDUSE=2;
    @DmEnumDesc(name="过敏排除后直接使用（本次不需再皮试）",description="过敏排除后直接使用（本次不需再皮试）")
	public static final Integer DIRECTUSEEXCLUDEALLERGIC=4;
    @DmEnumDesc(name="过敏皮试结果未出",description="过敏皮试结果未出，需提示并做人工选择")
	public static final Integer WAITSKINTESTRPT=5;
    @DmEnumDesc(name="未知",description="未知")
	public static final Integer UNKNOWN=9;
}	

File: client/iih_stub/iih.ci_stub/iih.ci.ord/pub/d/AllergicPharmHandleMode.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.ComponentModel;

namespace iih.ci.ord.pub.d
{
   public enum AllergicPharmHandleMode {
	/// <summary>
	/// 过敏用药禁用
	/// </summary>
    [Description("过敏用药禁用")]
    ALLERGICPHARMFORBIDDEN=0,
	/// <summary>
	/// 过敏再皮试
	/// </summary>
    [Description("过敏再皮试")]
    ALLERGICRESKINTEST=1,
	/// <summary>
	/// 皮肤过敏试验
	/// </summary>
    [Description("皮肤过敏试验")]
    SKINALLERGICTEST=3,
	/// <summary>
	/// 过敏强制使用
	/// </summary>
    [Description("过敏强制使用")]
    ALLERGICFORCEDUSE=2,
	/// <summary>
	/// 过敏排除后直接使用（本次不需再皮试）
	/// </summary>
    [Description("过敏排除后直接使用（本次不需再皮试）")]
    DIRECTUSEEXCLUDEALLERGIC=4,
	/// <summary>
	/// 过敏皮试结果未出
	/// </summary>
    [Description("过敏皮试结果未出")]
    WAITSKINTESTRPT=5,
	/// <summary>
	/// 未知
	/// </summary>
    [Description("未知")]
    UNKNOWN=9,
   }
}
